index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title> 
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- 네비게이션 -->
+    <header class="nav-bar">
+      <a href="'/'" target="_blank">
+        <img  class="logo" src="Property 1=sm.png">
+      </a>
+      <a href="/login" target="_blank">
+        <img class="login-btn" src="btn_small.png">
```

Comment:
❗️ 수정요청
가능하다면 최대한 html, css로 구현해주세요!

```suggestion
        로그인
```